### PR TITLE
Automatically run tests

### PR DESCRIPTION
**Pull request by fearphage**
Does what it says on the tin.

### CODE REVIEW

**Comment by fearphage** [comment]
The following 2 tests have never passed in CI: 

* `difftest/tests/acl_works_flag`
* `difftest/tests/mysql_multiple_row_select`

I'm not sure what the solution is here. /cc @igroff @mikemartin1090

**Comment by fearphage** [comment]
Now down (up?) to 14 failed tests using the correct node version:

```shell
14 tests failed:
- /home/runner/work/epiquery2/epiquery2/difftest/tests/1000_rows_render_echo
- /home/runner/work/epiquery2/epiquery2/difftest/tests/acl_works_flag
- /home/runner/work/epiquery2/epiquery2/difftest/tests/file
- /home/runner/work/epiquery2/epiquery2/difftest/tests/mssql_bad_syntax
- /home/runner/work/epiquery2/epiquery2/difftest/tests/mssql_echo_mustache
- /home/runner/work/epiquery2/epiquery2/difftest/tests/mysql_bad_syntax
- /home/runner/work/epiquery2/epiquery2/difftest/tests/mysql_multiple_row_select
- /home/runner/work/epiquery2/epiquery2/difftest/tests/mysql_servername
- /home/runner/work/epiquery2/epiquery2/difftest/tests/render_echo_dot
- /home/runner/work/epiquery2/epiquery2/difftest/tests/render_only
- /home/runner/work/epiquery2/epiquery2/difftest/tests/render_only_multiple_times
- /home/runner/work/epiquery2/epiquery2/difftest/tests/ws_mssql_escaping
- /home/runner/work/epiquery2/epiquery2/difftest/tests/ws_mysql_multiple_simple_select
- /home/runner/work/epiquery2/epiquery2/difftest/tests/ws_mysql_servername
```

File: docker-compose.yaml [hunk 1]
```diff
@@ -11,29 +11,29 @@ services:
...
-      - ACCEPT_EULA=Y 
+      - ACCEPT_EULA=Y
       - SA_PASSWORD=YourV3ryStrong!Passw0rd
     networks:
-      - local 
+      - local
 
   sfdc:
-    build: ./mock-salesforce
+    image: glgresearch/sfdc-with-drakov:e04eea15
```
Comment (mikemartin1090) [inline docker-compose.yaml:28]: we pin this here, but not in our GitHub workflow

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12.3.1'
+      - name: Update hosts file
+        run: echo '127.0.0.1 mssql mysql sfdc' | sudo tee --append /etc/hosts
+      - name: pull images
+        run: |
+          docker pull glgresearch/sfdc-with-drakov
```
Comment (mikemartin1090) [inline .github/workflows/tests.yml:17]: not pinned - should it be?

File: docker-compose.yaml (hunk 1 above)
Comment (fearphage) [reply docker-compose.yaml:28]: Added it.

**Comment by mikemartin1090** [comment]
Child of: https://github.com/glg/metadevops-issues/issues/393

**Pull request merged**